packages/sdk/test/ci/credentialDeployment.test.ts

```diff
@@ -29,6 +44,92 @@ function createCredentialInput(revealedAttributes: AttributeKey[], idObject: Ide
+    expect(serializedTx.toString('hex')).toEqual(expected);
+});
+
+test('test create + deserialize v0 credentialDeployment', async () => {
```

Comment:
This was migrated from `../client/credentialDeployment.test.ts` to be included in CI.